frontend/src/data/quizzes.ts

```diff
@@ -34,12 +35,14 @@ const quizzes = [
     name: 'Quiz 1',
     languages: ['Python', 'Javascript'],
     codeSmells: ['Comment', 'Incorrect typing'],
+    time: 10,
   },
   {
     id: '2',
     name: 'Quiz 2',
     languages: ['Ruby', 'Elixir'],
     codeSmells: ['Typo', 'Logic Error'],
+    time: 300,
```

Comment:
I'm wondering if we want to make the `time` optional 🤔 Maybe we should ask the client.